plutus-ledger-constraints/plutus-ledger-constraints.cabal

```diff
@@ -44,6 +44,7 @@ library
     build-depends:
         base >=4.9 && <5,
         aeson -any,
+        cardano-api -any,
```

Comment:
We don't actually need `cardano-api`, do we?

Reply:
Yep, removed, thanks!